gradle.properties

```diff
@@ -1,14 +1,14 @@
 #Uncomment and set to the path of the Gradle plugins if they are not in niagara_home
 #gradlePluginHome=
 #The path to the installation of Niagara you are building against
-niagara_home=c:\\niagara\\niagara-4.13.0.186
+niagara_home=c:\\Niagara\\Niagara-4.14.0.162
```

Comment:
Sorry about this, it looks like these exercises didn't get updated to 4.14. I'll fix these when I get a chance.